tests/cypress/e2e/actions_tasks3/case_105_cloud_storage.js

```diff
@@ -25,11 +25,23 @@ context('Cloud storage.', () => {
         projectID: 'Some ID',
     };
 
+    const serverHost = Cypress.config('baseUrl').includes('3000') ? 'localhost' : 'minio';
```

Comment:
It has to be configured in cypress.config.js (maybe something like `minioHostname`)